src/grid/gridColumn.js

```diff
@@ -6,10 +6,9 @@ class GridColumnComponent extends React.Component {
         return !isEqual(this.props.children, nextProps.children);
     }
     render() {
-        const flexSize = this.props.flexGrow || '1';
-        const style = { flex: `${flexSize} 0 0px` };
+        const style = { flex: '1 0 0px' };
```

Comment:
why would we consider deprecating the ability for the parent to set flex-grow of a gridColumn?